test/Api.Test/Controllers/CollectionsControllerTests.cs

```diff
@@ -210,7 +210,7 @@ await sutProvider.GetDependency<IDeleteCollectionCommand>()
 
 
     [Theory, BitAutoData]
-    public async Task DeleteMany_UserCanNotAccessCollections_FiltersOutInvalid(Guid orgId, User user, Collection collection1, Collection collection2, SutProvider<CollectionsController> sutProvider)
+    public async Task DeleteMany_UserCanNotAccessCollections_FiltersOutInvalid(Guid orgId, Collection collection1, Collection collection2, SutProvider<CollectionsController> sutProvider)
```

Comment:
Is this test still accurate? I think the auth handler will throw if the user doesn't have access to all collections requested, it doesn't filter as described in the test name.

Reply:
Good call! Removed since the handler handles that and the test above handles invalid permissions.